### DISCUSSION TITLE
Make shop purchased TMogs giftable

### DISCUSSION THREAD

**Discussion by TelumehtarCC** [chromiecraft/chromiecraft#7649] [Suggestions]
Hi, simple suggestion. Similar to how we can gift pets, mounts, etc. Let's add TMogs to that capability please!

**Comment by TelumehtarCC** [comment]
![image](https://github.com/user-attachments/assets/d042ac68-4996-441b-814a-1c264450cef5)

No field to type in character name.

**Comment by Helias** [comment]
Thank you for the suggestion! I've made it suitable as a gift.
